packages/vm-config/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@agoric/vm-config",
+  "version": "0.1.0",
+  "description": "Config to start an Agoric VM",
+  "type": "module",
+  "main": "./index.js",
+  "repository": "https://github.com/Agoric/agoric-sdk",
+  "scripts": {
+    "build": "exit 0",
+    "test": "ava",
+    "lint-fix": "yarn lint:eslint --fix",
+    "lint": "run-s --continue-on-error lint:*",
+    "lint:types": "tsc -p jsconfig.json",
+    "lint:eslint": "eslint ."
```

Comment:
Our [docs on making a new package](https://github.com/Agoric/agoric-sdk/blob/master/README.md#adding-a-new-package) say:
> populate a new `packages/zoe/package.json`, using other packages as a template

Is there any more to it than that? I'm sometimes overwhelmed wondering which other packages to use as a template.

Reply:
Those docs are a little stale. e.g. we don't have to manually add to workspaces now because it has a globl.
I'll try to update in this PR.

as a template I used `boot/packages.json` (i actually copied the whole dir then cut it down)

Reply:
separated https://github.com/Agoric/agoric-sdk/pull/8142